plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerSchema.java

```diff
@@ -414,16 +414,20 @@ public JDBCStatement prepareLookupStatement(@NotNull JDBCSession session, @NotNu
 
         @Override
         protected SQLServerTableBase fetchObject(@NotNull JDBCSession session, @NotNull SQLServerSchema owner, @NotNull JDBCResultSet dbResult) {
+            String name = JDBCUtils.safeGetString(dbResult, "name");
+            if (CommonUtils.isEmpty(name)) {
+                return null;
```

Comment:
Please add logging about null db name

Reply:
Logging for tables and databases empty names added